sim/priest/swd.go

```diff
@@ -0,0 +1,65 @@
+package priest
+
+import (
+	"time"
+
+	"github.com/wowsims/tbc/sim/core"
+	"github.com/wowsims/tbc/sim/core/stats"
+)
+
+const SpellIDSWD int32 = 32996
+
+var SWDCooldownID = core.NewCooldownID()
+
+func (priest *Priest) newSWDTemplate(sim *core.Simulation) core.SimpleSpellTemplate {
+	baseCast := core.Cast{
+		Name:           "Shadow Word: Death",
+		CritMultiplier: 1.5,
+		SpellSchool:    stats.ShadowSpellPower,
+		Character:      &priest.Character,
+		BaseManaCost:   309,
+		ManaCost:       309,
+		CastTime:       0,
+		Cooldown:       time.Second * 12,
+		ActionID: core.ActionID{
+			SpellID:    SpellIDSWD,
+			CooldownID: SWDCooldownID,
+		},
+	}
+
+	effect := core.SpellHitEffect{
+		SpellEffect: core.SpellEffect{
+			DamageMultiplier:       1,
+			StaticDamageMultiplier: 1,
+		},
+		DirectInput: core.DirectDamageSpellInput{
+			MinBaseDamage:    672,
+			MaxBaseDamage:    664,
+			SpellCoefficient: 0.429,
+		},
+	}
+
+	// TODO: Manage health with SW:D or assume healers got you?
```

Comment:
I think its clear we're ignoring health concerns for the purposes of the sim, just delete this comment. We're definitely not going to limit warlock lifetaps so shouldnt do anything here.